store_monitoring/crud.py

```diff
@@ -41,4 +51,15 @@ def insert_bq_results(db):
             db.commit()
             db.refresh(data)
     return
-        
\ No newline at end of file
+
+def time():
```

Comment:
Rename the function. It doesn't match with what it's doing

Reply:
ok